balboa-client-jms/src/main/java/com/blist/metrics/impl/queue/MetricJmsQueue.java

```diff
@@ -64,7 +64,15 @@ private MetricJmsQueue(ConnectionFactory factory, String queueName) throws JMSEx
         this.factory = factory; // badness happens if it's GC'd
         Connection connection = factory.createConnection();
         realInstance = new MetricJmsQueueNotSingleton(connection, queueName);
-        realInstance.start();
+    }
+
+    @Override
+    public void close() throws Exception {
+        // TODO When a singleton object has two underlying implementations... that is a bad smell.
```

Comment:
Random TODOs like this are code litter. They so rarely get addressed.

What is bad? A singleton can wrap many underlying objects, that's pretty normal.

The weird thing is naming a class MetricJmsQueueNotSingleton. It's also weird that this instance method closes the static class member. That's definitely weird. Is it because the Java compiler won't let you have a static close method since there is a member method?